docs/src/catalyst_applications/ode_simulation_performance.md

```diff
@@ -0,0 +1,318 @@
+
+Finally, we should note that this is not an exact science, and sometimes explicit solvers can successfully solve a stiff problem. E.g. if we change the parameters values of our previous Brusselator model to `[:A => 1.0, :B => 4.0]`, the `Tsit5` will successfully be able to simulate it.
+
+
+## [ODE solver selection](@id ode_simulation_performance_solvers)
+OrdinaryDiffEq implements an unusually large number of ODE solvers, with the performance of the simulation heavily depending on which one is chosen. These are provided as the second argument to the `solve` command, e.g. here we use the `Tsit5` solver to simulate a simple production/degradation loop.
+```@example ode_simulation_performance_2
+using Catalyst, OrdinaryDiffEq
+
+rn = @reaction_network begin
+    (p,d), 0 <--> X
+end
+oprob = ODEProblem(rn, [:X => 0.1], (0.0,1.0), [:p => 1.0, :d => 0.2])
+solve(oprob, Tsit5())
+nothing # hide
+``` 
+Alternatively, if the full DifferentialEquations.jl package (rather than the OrdinaryDiffEq subpackage) is loaded, no solver is required (as one is automatically selected):
+```@example ode_simulation_performance_2
+using DifferentialEquations
+solve(oprob)
+nothing # hide
+``` 
+While the default choice is typically enough for most single simulations, if performance is important, it can be worthwhile exploring the available solvers to find one that is especially suited for the given problem. A complete list of possible ODE solvers, with advice on optimal selection, can be found [here](https://docs.sciml.ai/DiffEqDocs/stable/solvers/ode_solve/). This section will give some general advice.
+
+The most important part of solver selection is to select one appropriate for [the problem's stiffness](@ref ode_simulation_performance_stiffness). Generally, the `Tsit5` solver is recommended for non-stiff problems, and `Rodas5P` for stiff problems. For large stiff problems (with many species), `QNDF` can be a good choice. We can illustrate the impact of these choices by simulating our production/degradation model using the `Tsit5`, `BS5` (an explicit solver yielding [low error in the solution](@ref ode_simulation_performance_error)), `Rodas5P`, and `QNDF` solvers (benchmarking their respective performance using [BenchmarkTools.jl](https://github.com/JuliaCI/BenchmarkTools.jl)):
```

Comment:
`FBDF` instead of `QNDF`. Say a cutoff of around 50.

Instead of BS5, show Vern7.

Reply:
updated